lib/smb2.js

```diff
@@ -100,3 +100,7 @@ proto.mkdir = SMB2Connection.requireConnect(require('./api/mkdir'));
 
 proto.ensureDir = SMB2Connection.requireConnect(require('./api/ensureDir'));
 proto.getSize = SMB2Connection.requireConnect(require('./api/getSize'));
+
+proto.open = SMB2Connection.requireConnect(require('./api/open'));
```

Comment:
The requireConnect may schedule an auto-close so subsequent reads and/or closeFile may crash, unless connection.autoCloseTimeout is 0.

Reply:
Yes, I should put a note in the README.